ant-cli/src/commands.rs

```diff
@@ -70,6 +70,12 @@ pub enum FileCmd {
         /// Upload the file as public. Everyone can see public data on the Network.
         #[arg(short, long)]
         public: bool,
+        /// Skip creating an archive after uploading a directory.
+        /// When uploading a directory, files are normally grouped into an archive for easier management.
+        /// This flag uploads the files individually without creating the archive metadata.
+        /// Note: This option only affects directory uploads - single file uploads never create archives.
```

Comment:
> single file uploads never create archives

the PR description or help doc of the opt both indicates `no-archive` is a flag of an optional archive.
i.e. even when upload single file, there could be `archive` created.

at least one of the sides is incorrect and need to be corrected/synced.